js/repl/PluginConfig.js

```diff
@@ -58,20 +112,26 @@ const replDefaults: ReplState = {
   isPresetsTabExpanded: false,
   isSettingsTabExpanded: true,
   lineWrap: true,
+  presets: "es2015,react,stage-2",
```

Comment:
Nit: why was this moved? I kinda prefer this being alpha sorted :) (no big deal)

Reply:
Oh, I just merged crookedly. Will fix, thanks :)